databricks/koalas/tests/test_indexes.py

```diff
@@ -1001,3 +1001,64 @@ def test_unique(self):
 
         self.assert_eq(kmidx.unique().sort_values(), pmidx.unique().sort_values())
         self.assert_eq(kmidx.unique().sort_values(), pmidx.unique().sort_values())
+
+    def test_union(self):
+        # Index
+        pidx1 = pd.Index([1, 2, 3, 4])
+        pidx2 = pd.Index([3, 4, 5, 6])
+        kidx1 = ks.from_pandas(pidx1)
+        kidx2 = ks.from_pandas(pidx2)
+
+        self.assert_eq(kidx1.union(kidx2).sort_values(), pidx1.union(pidx2).sort_values())
```

Comment:
The values must be sorted by default, do we need `sort_values()`?

Reply:
Ah, I think that's my mistake.

Maybe we should use `False` as the default value unlike pandas since the cost issue, right?

Reply:
If we have `sort` argument (or something the same meaning), we usually follow pandas behavior.

Reply:
@ueshin Ah, okay i got it. I can manage this.

Thanks for the comment :)